app/src/main/java/com/jerboa/model/CommunityViewModel.kt

```diff
@@ -277,6 +290,62 @@ class CommunityViewModel(account: Account, communityArg: Either<CommunityId, Str
         }
     }
 
+    override fun getNextPost(current: PostId?, account: Account?): PostId? {
+        val res = postsRes
+        return if (res is ApiState.Success) {
+            if (current == null) {
+                res.data.posts.firstOrNull()?.post?.id
+            } else {
+                res.data.posts
+                    .mapIndexed { index, postView -> index to postView }
+                    .firstOrNull { it.second.post.id == current }
+                    ?.first?.let { currIndex ->
+                        if (currIndex >= res.data.posts.size - 7) {
```

Comment:
This 7 is a buffer of next posts loaded already? Ifso make it constant somewhere that explains that

Reply:
Yes, it's intended to ensure that there are posts loaded before the user swipes left - otherwise the swipe will have no effect, which leads to a poor experience when you hit the end of a page (you end up having to swipe twice)

Will pull out as a const.